docker/dev/setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+source config.sh
+set -e
+
+apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install $DEV_APT_PACKAGES
+
+install_geosupport
+
+python3 -m pip install -e ./dcpy -r requirements.txt
```

Comment:
@fvankrieken Maybe I don't quite follow... But shouldn't we be doing an editable install of ./dcpy on the version in the mounted volume? 

In general, love this approach. Would allow us to stop futzing with the python path just to include dpcy

Reply:
Yeah, good catch I was literally about to comment on this line that this is the one that maybe is a little off.

Mainly because 
1) you are right, editable install here doesn't really make sense in terms of mechanism, and
2) this is the "complete" image for now... i.e. the one that also gets used for tests. So ideally it'd just work out of the box, but actually as I write this, tests should be using the dcpy in the codebase anyways

So it seems like as a solution
1) don't install here
2) install as part of "docker setup script" that currently gets run as first step of many jobs

Per quick irl discussion, going to do the same for build containers